### PR TITLE
Mutations/update todo status

### LINKED ISSUE #17: Change Todo Status
Acceptance Criteria
- [ ] A checkbox to change the status 
- [ ] The UI should change
- [ ] The status of the todo should toggle

### PR DESCRIPTION
**Pull request by harshil1712**
Fixes #17

### CODE REVIEW

**Comment by SiddharthSham** [comment]
@kassens please review

File: src/App.js [hunk 1]
```diff
@@ -1,52 +1,88 @@
...
+
+  const handleStatusChange = () => {
+    markAllTodos({
+      variables: {
+        TodoStatus: {
+          complete: !allStatus,
+        },
+      },
+      onCompleted: (data) => {
+        console.log(data)
```
Comment (jevakallio) [inline src/App.js:37]: If we leave console.log statements for success cases, make sure you label them so people viewing the logs know where they're coming from.

I would personally remove this but ok to keep if it's necessary.

File: src/App.js [hunk 2]
```diff
@@ -1,52 +1,88 @@
...
+        <div className="top-bar">
+          <div className="menu">
+            <div>
+              <input
+                type="checkbox"
+                id="markAllCompleted"
+                checked={allStatus}
+                onChange={handleStatusChange}
+              />
+              <label htmlFor="markAllCompleted">Mark 13 as done! 🎉</label>
```
Comment (jevakallio) [inline src/App.js]: Hardcoded 13?

File: src/App.js [hunk 3]
```diff
@@ -1,52 +1,88 @@
...
+                value="active"
+              ></input>
+              <label htmlFor="active">todo</label>
+              <input
+                type="radio"
+                id="completed"
+                name="view"
+                value="completed"
+              ></input>
+              <label htmlFor="completed">7 done 💪</label>
```
Comment (jevakallio) [inline src/App.js]: Hardcoded 7?

File: src/App.js (hunk 2 above)
Comment (harshil1712) [reply src/App.js]: We have updated this on another branch. It was just a placeholder value for the time being.

File: src/App.js (hunk 3 above)
Comment (harshil1712) [reply src/App.js]: We have updated this on another branch. It was just a placeholder value for the time being.

**Pull request merged**